### ISSUE TITLE
add-on does not install on NVDA 2025.3.2

### ISSUE THREAD

**Issue by a-singer** [jamalmazrui/HomerView#4]
Thanks for producing this, it looks extraordinary. I am not, though, able to get it installed on NVDA. I am not as familiar as I should be with the log, but I think the below shows the problem. If it doesn't, or if you need more information, feel free to let me know. This is running on Windows 10.

Error importing global plugin homerView
Traceback (most recent call last):
  File "globalPluginHandler.pyc", line 23, in listPlugins
  File "importlib\__init__.pyc", line 126, in import_module
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\test\AppData\Roaming\nvda\addons\homerView\globalPlugins\homerView\__init__.py", line 42, in <module>
    from .pageBuffer import HomerViewBuffer, HomerViewDocument, bindFallbackGestures
  File "C:\Users\test\AppData\Roaming\nvda\addons\homerView\globalPlugins\homerView\pageBuffer.py", line 50, in <module>
    from .service import service
  File "C:\Users\test\AppData\Roaming\nvda\addons\homerView\globalPlugins\homerView\service.py", line 41, in <module>
    from . import pageExplorer
  File "C:\Users\test\AppData\Roaming\nvda\addons\homerView\globalPlugins\homerView\pageExplorer.py", line 363
    f"{asList(dPage.get("autoplay"))} media {plural(asList(dPage.get("autoplay")), 'element')} "
                         ^^^^^^^^
SyntaxError: f-string: unmatched '('
